README.md

```diff
@@ -18,15 +18,20 @@ Guidance on onboarding samples to docs.microsoft.com/samples: https://review.doc
 Taxonomies for products and languages: https://review.docs.microsoft.com/new-hope/information-architecture/metadata/taxonomies?branch=master
 -->
 
-Give a short description for your sample here. What does it do and why is it important?
+This is a replica of the Urlist website but re-architected to use server based backend hosted in Azure AppService instead of being serverless. The original project can be found here https://github.com/burkeholland/the-urlist
+
+Please [check here](resources/infrastructure.md) to view and read about the new architecture.

```

Comment:
I would create a top level `docs` folder instead of `infrastructure`.  This is more common in git repos.